Sources/MapboxCoreNavigation/NavigationSettings.swift

```diff
@@ -187,3 +187,18 @@ extension String {
         return "MB" + self
     }
 }
+
+extension MeasurementSystem {
+    /// Converts `MapboxDirections.MeasurementSystem` into `LengthFormatter.Unit`.
+    public var distanceUnit: LengthFormatter.Unit {
```

Comment:
Not sure if we need to expose this to public or hide under "spi".

Reply:
`MeasurementSystem` is defined by MapboxDirections; these conversions should ultimately live there too, so I’d suggest hiding them for now using `:nodoc:`.